DRAFT_functional plant indicators_wetland.Rmd

```diff
@@ -961,21 +879,19 @@ wetland.ref.cov.val <- data.frame(N1=rep('wetland',(nrow(tab)*2*indEll.n)),
 
 wetland.ref.cov.val
 wetland.ref.cov.val$grunn <- as.factor(wetland.ref.cov.val$grunn)
+wetland.ref.cov.val$Ind <- as.factor(wetland.ref.cov.val$Ind)
 summary(wetland.ref.cov.val)
 
 
-
-#write.table(wetland.ref, file='output/ref_for_scaling/mount_ref_ANO.txt',quote=FALSE,sep="\t",col.names=TRUE,row.names=FALSE,dec=".")
-
-
-
-
 ```
 
+```{r}
+head(wetland.ref.cov.val)
+```
 
-Once testing data (ANO), and reference data (NiN) are in place, we can calculate CWMs of the selected indicators for the ANO-community data and scale them against the scaling values from the reference distribution.
-
+Once testing data (ANO), and the scaling values from the reference data (NiN) are in place, we can calculate CWMs of the selected indicators for the ANO-community data and scale them against the scaling values from the reference distribution. Note that we scale each ANO-plot's CWM against either the lower threshold value and the min value OR the upper threshold value and the max value based on whether the CWM is smaller or higher than the reference value. Since the scaled values for both sides range between 0 and 1, we generate separate lower and upper indicators for each plant functional indicator type. An ANO plot can only have a scaled value in either the lower or the upper indicator (the other one will be 'NA'), except for the unlikely event that the CWM exactly matches the reference value, in which case both lower and upper indicator will receive a scaled indicator value of 1.
```

Comment:
it is 2 indicators for each plant functional indicator, yes...a lower and an upper one. I can look over the text to make this more clear and explicit. It does not make sense to merge them together, because they indicate totally opposite environmental challenges. Low scores in the the lower moisture indicator indicate that it is too dry, while low scores in the upper moisture indicator indicate that it is too wet. If we simply merge them into one, we would not have any information on what a low score actually means, too dry or too wet.

The scaling function is written in a general way, to work in both directions, but in the actual scaling loop you'll see that the scaling is done separately for the values that are lower and higher than the reference value, respectively.